arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-morocco.dts

```diff
@@ -188,6 +188,24 @@
 	};
 };
 
+#ifdef LCD_ENABLE
```

Comment:
Can't use #ifdef. Both the drivers needed to be in memory. However, spi-txrx should be bind by default. In hpm_fpga_update.sh, we need to unbind the spi-txrx and bind flash driver just before updating the hpm fpga flash. After flash is done, unbind spi-flash driver and rebind spi-txrx similar to to how set-apml.sh is being done.